src/libYARP_dev/src/devices/msgs/CMakeLists.txt

```diff
@@ -12,111 +12,111 @@ yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/stateExt.thrift"
+#                 "${CMAKE_CURRENT_SOURCE_DIR}/ros"
+#                 ros_msg_sources
+#                 ros_msg_headers
+#                 ros_msg_include_dirs)
+# include(${CMAKE_CURRENT_SOURCE_DIR}/ros/geometry_msgs_Transform_msg.cmake)
+# 
+# yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/ros/msg_file/geometry_msgs/TransformStamped.msg"
+#                 "${CMAKE_CURRENT_SOURCE_DIR}/ros"
+#                 ros_msg_sources
+#                 ros_msg_headers
+#                 ros_msg_include_dirs)
+# include(${CMAKE_CURRENT_SOURCE_DIR}/ros/geometry_msgs_TransformStamped_msg.cmake)
+# 
+# yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/ros/msg_file/geometry_msgs/WrenchStamped.msg"
+#                 "${CMAKE_CURRENT_SOURCE_DIR}/ros"
+#                 ros_msg_sources
+#                 ros_msg_headers
+#                 ros_msg_include_dirs)
+# include(${CMAKE_CURRENT_SOURCE_DIR}/ros/geometry_msgs_WrenchStamped_msg.cmake)
+# 
+# yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/ros/msg_file/nav_msgs/OccupancyGrid.msg"
+#                 "${CMAKE_CURRENT_SOURCE_DIR}/ros"
+#                 ros_msg_sources
+#                 ros_msg_headers
+#                 ros_msg_include_dirs)
+# include(${CMAKE_CURRENT_SOURCE_DIR}/ros/nav_msgs_OccupancyGrid_msg.cmake)
+# 
+# yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/ros/msg_file/nav_msgs/MapMetaData.msg"
+#                 "${CMAKE_CURRENT_SOURCE_DIR}/ros"
+#                 ros_msg_sources
+#                 ros_msg_headers
+#                 ros_msg_include_dirs)
+# include(${CMAKE_CURRENT_SOURCE_DIR}/ros/nav_msgs_MapMetaData_msg.cmake)
+# 
+# yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/ros/msg_file/tf/tfMessage.msg"
+#                 "${CMAKE_CURRENT_SOURCE_DIR}/ros"
+#                 ros_msg_sources
+#                 ros_msg_headers
+#                 ros_msg_include_dirs)
+# include(${CMAKE_CURRENT_SOURCE_DIR}/ros/tf_tfMessage_msg.cmake)
+# 
+# yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/ros/msg_file/visualization_msgs/Marker.msg"
+#                 "${CMAKE_CURRENT_SOURCE_DIR}/ros"
+#                 ros_msg_sources
+#                 ros_msg_headers
+#                 ros_msg_include_dirs)
+# include(${CMAKE_CURRENT_SOURCE_DIR}/ros/visualization_msgs_Marker_msg.cmake)
+# 
+# yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/ros/msg_file/tf2_msgs/TFMessage.msg"
+#                 "${CMAKE_CURRENT_SOURCE_DIR}/ros"
+#                 ros_msg_sources
+#                 ros_msg_headers
+#                 ros_msg_include_dirs)
+# include(${CMAKE_CURRENT_SOURCE_DIR}/ros/tf2_msgs_TFMessage_msg.cmake)
+# 
+# yarp_idl_to_dir("${CMAKE_CURRENT_SOURCE_DIR}/ros/msg_file/visualization_msgs/MarkerArray.msg"
+#                 "${CMAKE_CURRENT_SOURCE_DIR}/ros"
+#                 ros_msg_sources
+#                 ros_msg_headers
+#                 ros_msg_include_dirs)
+# include(${CMAKE_CURRENT_SOURCE_DIR}/ros/visualization_msgs_MarkerArray_msg.cmake)
```

Comment:
Can we remove all this commented lines? If we ever need them, git remembers.
@drdanz

Reply:
Sure! Git remembers, I forgot! :laughing: